int64 cloud_id
---
sensor_msgs/PointCloud2 landmark_cloud